test/test_model_large.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import numpy as np
+import tensorflow as tf
+import onnx
+from onnx_tf.backend import run_model, prepare
+from onnx import helper
+from onnx.onnx_pb2 import TensorProto
+
+class TestNode(unittest.TestCase):
+  MODEL_PATH = "../../onnx_models/"
```

Comment:
Instead of assuming a path, can we download the models?

Reply:
yeah i think shell script is better here since we have to unpack a compressed file.